drivers/wifi/eswifi/eswifi_offload.c

```diff
@@ -615,6 +660,69 @@ static int eswifi_off_disable_bypass(struct eswifi_dev *eswifi)
 	return err;
 }
 
+void eswifi_offload_async_msg(struct eswifi_dev *eswifi, char *msg, size_t len)
+{
+	char msg_tcp_accept[] = "[TCP SVR] Accepted ";
+
+	if (!strncmp(msg, msg_tcp_accept, sizeof(msg_tcp_accept) - 1)) {
+		struct eswifi_off_socket *socket = NULL;
+		struct in_addr *sin_addr;
+		uint8_t ip[4];
+		uint16_t port = 0;
+		char *str;
+		int i = 0;
+
+		/* extract client ip/port e.g. 192.168.1.1:8080 */
+		/* TODO: use net_ipaddr_parse */
+		str = msg + sizeof(msg_tcp_accept) - 1;
+		while (*str) {
+			if (i < 4) {
+				ip[i++] = atoi(str);
+			} else if (i < 5) {
+				port = atoi(str);
+				break;
+			}
+			while (*str && (*str != '.') && (*str != ':')) {
```

Comment:
Please put empty lines after ending block } (in above)

Reply:
done